src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
```

Comment:
let's remove redundant empty lines, fix this in all classes

Reply:
done